### DISCUSSION TITLE
Welcome to MinimalCountdown Discussions!

### DISCUSSION THREAD

**Discussion by SKemenov** [SKemenov/MinimalCountdown#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by macinteractau** [comment]
Hi there, this is almost exactly what we need, but in our case we want to set a specific countdown on multiple devices, i.e. we would like to be able to preconfigure the screensaver before deploying it to multiple devices.
We have been dabbling a little bit, but could not get the following to work...
A. 'hard code' a specific time and date into the app
B. 'hard code' to only show date, hours and minutes, i.e. no seconds
C. and the icing on the cake to rotate between a set number of background images
...maybe you can give us some hints - we are completely rookies in swift / coding apps.
DS.

**Reply by SKemenov** [reply]
Hi there!
Maybe I can help you with the first two points, but at this moment, I'm not ready to add any background images to my screensaver. However, please give me more information:
1. What is the exact date and time you need to preset?
2. How often do you want to change this date?

As an option, I could create a custom install package for you if you're not going to change the preset date too often)

**Reply by SKemenov** [reply]
Hi! v2.0.0 is out and covers most of this:

A. Preconfigure across devices — settings now live in a plain file at `/Users/Shared/MinimalCountdown/settings.json`. Configure it on one Mac, copy that file to the others, and they'll all share the same saver settings.
B. Hide seconds — you can now pick exactly which units to show (Settings -> Appearance).
C. Rotating background images — still not on the roadmap; the saver stays focused on the minimal countdown.

Grab it here: [Release v2.0.0](https://github.com/SKemenov/MinimalCountdown/releases/tag/v2.0.0)

**Comment by Smitty20-20** [comment]
Hi, I'm trying to set a countdown date for July 2027, but everything beyond Feb 26, 2027 is greyed out.  Is this a known limitation? Is it possible to extend the maximum countdown date?  Thanks in advance for your assistance.

**Reply by SKemenov** [reply]
Hi! That was a one-year limit — fixed in v2.0.0. The selectable range now reaches up to 999 days before or after today, so July 2027 works fine now.

Grab it here: [Release v2.0.0](https://github.com/SKemenov/MinimalCountdown/releases/tag/v2.0.0)
